cli-tests/support/pmm-bin.setup.ts

```diff
@@ -0,0 +1,18 @@
+import { test as setup } from '@playwright/test';
+import * as cli from '@helpers/cliHelper';
+
+const oldImage = 'percona/pmm-server:2.32.0';
+const newImage = 'percona/pmm-server:2.33.0';
+
+setup.describe.configure({ mode: 'parallel' });
+setup(`pull ${newImage}`, async () => {
+  process.env.server_image = newImage;
+  const output = await cli.exec(`docker pull ${newImage}`);
+  await output.assertSuccess();
+});
+
+setup(`pull ${oldImage}`, async () => {
+  process.env.server_image_old = oldImage;
+  const output = await cli.exec(`docker pull ${oldImage}`);
+  await output.assertSuccess();
+});
```

Comment:
Let's not pull the images here. We want to make sure `pmm` cli can pull them properly.

Reply:
I cannot agree on given parallel run. Why would we want tests to download the same image in 6 thread same time?
I will ad a single test to check the download. It actually present - test which install defaults - it downloads the latest image.
Also please consider this as improvement point to un-hardcode pmm-server versions.

Reply:
afaik `docker` cli downloads the image only once, even with multiple concurrent requests to download the image.
It may not be the same case when using API but I did not notice the image being downloaded multiple times when running locally.

Reply:
I've did a research. Indeed docker downloads same image in the single thread for any amount of started to pull terminals.
Also downloading progress does not trash the logs.
These were 2 main reason to have pull before the tests. Removed.